floorsTest.json

```diff
@@ -0,0 +1,10 @@
+[
+  "rubiconBidAdapter",
+  "priceFloors",
+  "currency",
+  "prebidServerBidAdapter",
+  "rubiconAnalyticsAdapter",
+  "dfpAdServerVideo",
+  "s2sTesting",
+  "appnexusBidAdapter"
+]
```

Comment:
Will remove this file before making PR into prebid.

Just used for internal testing for now.